.github/workflows/e2e-tests-on-PR.yml

```diff
@@ -0,0 +1,52 @@
+name: Cypress
+
+env:
+  CYPRESS_VIDEOS: ./e2e-tests/cypress/videos
+  CYPRESS_SCREENSHOTS: ./e2e-tests/cypress/screenshots
+  CYPRESS_REPORTS: ./e2e-tests/cypress/cucumber-report
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
Do we want to also put these on fix/feature branches? Just thinking that we probably want these on the PR process so we don't end up with broken `master` branches

Reply:
we try with PR execution for now; see how it goes